tensorflow/java/src/test/java/org/tensorflow/OperationBuilderTest.java

```diff
@@ -149,6 +151,36 @@ public void setAttrShape() {
     }
   }
 
+  @Test
+  public void addControlInput() {
+    assertEquals(0, lessEqualControl(2, 3).size());
+    try {
+      lessEqualControl(3, 2);
```

Comment:
Nice! Thanks @asimshankar for the pointers to simplify the test, and the other suggestions. I've incorporated all of them in the next update of this PR.